worlds/kh1/Items.py

```diff
@@ -0,0 +1,534 @@
+    "Tinker Bell":                       KH1ItemData("Summons",          code = 264_5003, classification = ItemClassification.progression, max_quantity = 1,  weight = 10),
+    "Mushu":                             KH1ItemData("Summons",          code = 264_5004, classification = ItemClassification.progression, max_quantity = 1,  weight = 10),
+    "Simba":                             KH1ItemData("Summons",          code = 264_5005, classification = ItemClassification.progression, max_quantity = 1,  weight = 10),
+    "Progressive Fire":                  KH1ItemData("Magic",            code = 264_6001, classification = ItemClassification.progression, max_quantity = 3,  weight = 10),
+    "Progressive Blizzard":              KH1ItemData("Magic",            code = 264_6002, classification = ItemClassification.progression, max_quantity = 3,  weight = 10),
+    "Progressive Thunder":               KH1ItemData("Magic",            code = 264_6003, classification = ItemClassification.progression, max_quantity = 3,  weight = 10),
+    "Progressive Cure":                  KH1ItemData("Magic",            code = 264_6004, classification = ItemClassification.progression, max_quantity = 3,  weight = 10),
+    "Progressive Gravity":               KH1ItemData("Magic",            code = 264_6005, classification = ItemClassification.progression, max_quantity = 3,  weight = 10),
+    "Progressive Stop":                  KH1ItemData("Magic",            code = 264_6006, classification = ItemClassification.progression, max_quantity = 3,  weight = 10),
+    "Progressive Aero":                  KH1ItemData("Magic",            code = 264_6007, classification = ItemClassification.progression, max_quantity = 3,  weight = 10),
+   #"Traverse Town":                     KH1ItemData("Worlds",           code = 264_7001, classification = ItemClassification.progression, max_quantity = 1,  weight = 10),
+    "Wonderland":                        KH1ItemData("Worlds",           code = 264_7002, classification = ItemClassification.progression, max_quantity = 1,  weight = 10),
+    "Olympus Coliseum":                  KH1ItemData("Worlds",           code = 264_7003, classification = ItemClassification.progression, max_quantity = 1,  weight = 10),
+    "Deep Jungle":                       KH1ItemData("Worlds",           code = 264_7004, classification = ItemClassification.progression, max_quantity = 1,  weight = 10),
+    "Agrabah":                           KH1ItemData("Worlds",           code = 264_7005, classification = ItemClassification.progression, max_quantity = 1,  weight = 10),
+    "Halloween Town":                    KH1ItemData("Worlds",           code = 264_7006, classification = ItemClassification.progression, max_quantity = 1,  weight = 10),
+    "Atlantica":                         KH1ItemData("Atlantica",        code = 264_7007, classification = ItemClassification.progression, max_quantity = 1,  weight = 10),
```

Comment:
```suggestion
    "Atlantica":                         KH1ItemData("Worlds",        code = 264_7007, classification = ItemClassification.progression, max_quantity = 1,  weight = 10),
```

This was probably the intent here?

Reply:
This is intended to only shuffle in Atlantica if the setting is turned on

Reply:
You can't have an item that has the same name as an item_name_group, so you'll have to change something about how your item_name_groups are made

Reply:
Oh okay, no problem

Reply:
Pushed a quick fix just to change the item group name from "Atlantica" to "AL"